ckanext/dgu/forms/dataset_form.py

```diff
@@ -458,6 +458,11 @@ def to_list(key, data, errors, context):
     if isinstance(item, basestring):
         data[key] = [item]
 
+def commas_to_list(key, data, errors, context):
```

Comment:
@davidread what do you think of this? Would it be better to change the form so that there are multiple inputs for the secondary theme?